pkg/v1/tkg/client/init.go

```diff
@@ -768,3 +784,29 @@ func (c *TkgClient) safelyAddFeatureFlag(featureFlags map[string]string, feature
 	}
 	return featureFlags
 }
+
+// removeKappControllerLabelsFromClusterClassResources removes kapp-controller labels from clusterclass resources
+func (c *TkgClient) removeKappControllerLabelsFromClusterClassResources(regionalClusterClient clusterclient.Client) error {
+	errList := []error{}
+	labelsToBeDeleted := []string{"kapp.k14s.io/app", "kapp.k14s.io/association"}
```

Comment:
nit-ish: There are a lot of harcoded strings here (up to line 801). Can we either make these string constants or use existing constants?

Reply:
Updated the PR to use constants. Also updated few more places to use consistent constants across the board.